data_gen/old/Benchmark/CHANGES.txt

```diff
@@ -1 +0,0 @@
-TBD
```

Comment:
'old' folder was deleted. I looked into it and did not see the reason to keep it around. Let me know if you disagree.